assignments/closure.js

```diff
@@ -1,11 +1,19 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const fun = () => {
+  return ('this is an example of function scope aka local scope');
+}
```

Comment:
Thomas, this isn't a closure. A closure, most basically, is a function inside of another function. This is an example of scope, which closures take advantage of, but it isn't a closure. Let me know if you have any questions!